application/classes/Ushahidi/Repository/User.php

```diff
@@ -114,13 +134,18 @@ public function getSearchFields()
 	public function setSearchConditions(SearchData $search)
 	{
 		$query = $this->search_query;
+		$table = $this->getTable();
 
 		if ($search->q)
 		{
 			$query->and_where_open();
 			$query->where('email', 'LIKE', "%" . $search->q . "%");
 			$query->or_where('realname', 'LIKE', "%" . $search->q . "%");
 			$query->and_where_close();
+
+			// Adding search contacts
+			$query->join('contacts')->on("$table.id", '=', 'contacts.user_id')
+			->or_where('contacts.contact', 'like', '%' . $search->q . '%');
```

Comment:
@willdoran it occurs to me now that we should probably limit searching by email or contact to users with permissions, potentially you could ID a user via search despite the fact the API doesn't return their email or contact info.

Reply:
True we should add a check, I'll patch it later today.